resources/devbox/macos/keyman.macos.env.sh

```diff
@@ -5,18 +5,18 @@ export ANT_HOME=/usr/local/opt/ant
 export MAVEN_HOME=/usr/local/opt/maven
 export GRADLE_HOME=/usr/local/opt/gradle
 export ANDROID_HOME=~/.android
-export ANDROID_SDK_ROOT="$HOME/Library/Android/sdk"
 
 export PATH=$ANT_HOME/bin:$PATH
 export PATH=$MAVEN_HOME/bin:$PATH
 export PATH=$GRADLE_HOME/bin:$PATH
 export PATH=$ANDROID_HOME/tools:$PATH
 export PATH=$ANDROID_HOME/tools/bin:$PATH
 export PATH=$ANDROID_HOME/platform-tools:$PATH
+export PATH=$ANDROID_HOME/cmdline-tools/latest/bin:$PATH
 export PATH=$ANDROID_HOME/build-tools/30.0.3:$PATH
-export PATH="/usr/local/opt/openjdk@8/bin:$PATH"
+export PATH="/opt/homebrew/opt/java11/bin:$PATH"
 export PATH="$HOMEBREW_PREFIX/opt/coreutils/libexec/gnubin:$PATH"
-export JAVA_HOME=/usr/local/opt/openjdk@8
+export JAVA_HOME=/opt/homebrew/opt/java11
```

Comment:
Potential caveat:  if I'm not mistaken, this path for things installed via Homebrew may be Apple Silicon specific.  Granted, Mac in general is headed that way, so maybe it's not a bad thing to do this, anyway.

Might be worth a spot of discussion if I'm right about the distinction, though.

Reply:
That's right, for Intel Macs, Homebrew was in /usr/local/opt, and it uses /opt/homebrew/opt for Apple Silicon. I still use an Intel machine for my dev machine, but we should be on all Apple Silicon before too long.

Reply:
If we feel it important, I can _probably_ make this adaptive: https://stackoverflow.com/a/65259353

It is technically possible to detect `i386` vs `arm` in bash shell.  Seems to hold in zsh too.  That said, if a process in the chain is being run via Rosetta, _that_ will cause an `i386` report, according to some of the comments elsewhere in that StackOverflow thread.  May need to take care.

Reply:
Just test if the arm path exists, and if not, fallback to the i386 path.